rebel-readline/src/rebel_readline/clojure/utils.clj

```diff
@@ -24,7 +24,7 @@
     (.startsWith (str ns) "clojure.")
     (cond-> "https://clojuredocs.org/"
       ns (str ns)
-      name (str "/" name))
+      name (str "/" (string/replace name #"\?" "_p"))
```

Comment:
The regex here would be better as `#"\?$"`.

Reply:
I can see that, considering that the documented functions are all predicates that actually end with `?`.